### PR TITLE
Removed season supporters

### PR DESCRIPTION
**Pull request by EliAndrewC**
Season Supporters are a MAGFest-specific feature which aren't even shared by other MAG events.  Therefore, they really don't belong in the core repo, and as such they've been moved to the ``magprime`` repo.  I'll link to the PR which adds them there in a comment.

### CODE REVIEW

File: development-defaults.ini [hunk 1]
```diff
@@ -176,20 +176,4 @@ group  = "Taking an unassigned Group badge (group leader must be present)"
 staff_support = "Jack Boyd"
 security = "The Dorsai Irregulars"
```
Comment (EliAndrewC) [inline development-defaults.ini]: This is moved into the ``configspec.ini`` of the ``magprime`` plugin.

File: uber/automated_emails.py [hunk 2]
```diff
@@ -8,6 +8,8 @@
 class AutomatedEmail:
     instances = OrderedDict()
```
Comment (EliAndrewC) [inline uber/automated_emails.py]: There were some places where we were special-casing the ``PrevSeasonSupporter`` model.  This is now genericized.

**Comment by EliAndrewC** [comment]
This PR must be merged before https://github.com/magfest/magprime/pull/5 is merged, which is noted in that PR.

File: uber/automated_emails.py (hunk 2 above)
Comment (binary1230) [inline uber/automated_emails.py]: nice

**Comment by binary1230** [comment]
+1

**Pull request merged**